src/sonic-yang-models/yang-models/sonic-static-route.yang

```diff
@@ -0,0 +1,124 @@
+module sonic-static-route {
+  yang-version 1.1;
+  namespace "http://github.com/Azure/sonic-static-route";
+  prefix sroute;
+
+  import sonic-vrf {
+    prefix vrf;
+  }
+  import ietf-inet-types {
+    prefix inet;
+  }
+  import sonic-types {
+    prefix stypes;
+  }
+
+  organization
+    "SONiC";
+  contact
+    "SONiC";
+  description
+    "STATIC ROUTE yang Module for SONiC OS";
+
+  revision 2022-03-17 {
+    description
+      "First Revision";
+  }
+
+  container sonic-static-route {
+    container STATIC_ROUTE {
+      description
+        "STATIC_ROUTE part of config_db.json";	
+	list STATIC_ROUTE_TEMPLATE_LIST {
+        key "prefix";
+        leaf prefix {
+          type inet:ip-prefix;
+          description
+            "prefix is the destination IP address, as key";
+        }
+        leaf nexthop {
+          type string;
+          description
+            "The next-hop that is to be used for the
+             static route as IP address. When interface needs to be
+             specified, use 0.0.0.0 as leaf value";
+        }
+        leaf ifname {

```

Comment:
can you use leafref?

Reply:
Leafref cannot be used as these are comma separated values to support ECMP.

Reply:
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-ntp.yang#L52-L79

Can you clarify the difference between comma separated interface and leaf-list?

Reply:
https://github.com/sonic-net/sonic-buildimage/pull/11499 has the same changes (got committed without the files during pulling to latest in github). 
Comma separated was added as part this PR, which would be of the format - interfaces will be "Ethernet0, Ethernet4"